cli/src/test/debug.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+## source "$(dirname "${0}")"/functions-test
+
+echo my PWD is: $PWD
```

Comment:
This file does not seem to be used anywhere...
Was it included by accident as a leftover from local debugging?
Can we remove it?

Reply:
Removed now.